Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6
+RUN set -x && \
+	pip install nornir
+VOLUME [ "/nornir" ]
```

Comment:
I don't think exposing this volume adds any benefit. Same for the workdir

Reply:
The workdir should (ideally) be set to the same directory as where you mount your scripts.
Obiously this can be configured using the docker cli, but I thought adding some sane defaults made sense. Both can be overridden if a user wants too.